init.sh

```diff
@@ -19,7 +21,16 @@ deps=(
 )
 
 mkdir root
-rsync -a sources/ configs/ build.sh root/build
+rsync -avP sources/ configs/ build.sh config.sh root/build
+
+cd root/build
+echo "Extracting mrustc-${mrustc_version}"
+tar xzf mrustc-${mrustc_version}.tar.gz
+for v in ${mrustc_rustc_version} "${rustc_versions[@]}" ; do
+    echo "Extracting rustc-$v"
+    tar xzf rustc-$v-src.tar.gz -C rustc-$v --strip-components 1
+done
```

Comment:
It looks like this ends up invoking the next commands in the wrong directory (root/build) but I fixed it in add27c4bbf0633c201f6b06195c6740f83c4ba1e.